semantic_conventions/lib/opentelemetry/semantic_conventions/version.rb

```diff
@@ -6,6 +6,6 @@
 
 module OpenTelemetry
   module SemanticConventions
-    VERSION = '1.10.0'
+    VERSION = '1.10.1'
```

Comment:
The autobump was weird here: 1.10.0 -> 1.8.1. Is this supposed to match the semantic conventions version (in the spec) or just be a "normal" gem version number?

Reply:
I think the bump was supposed to be 1.8.0 -> 1.8.1. (The current released version is 1.8.0: see https://rubygems.org/gems/opentelemetry-semantic_conventions).

Reply:
Sorry - GitHub weirdness meant I missed your comments, added my own and resolved the conversation 🤦 - sorry about that!

What do we prefer here? Do we want the semantic conventions gem version to match the semconv spec version used to generate it, or should we stick with the normal version bumping behaviour?

Reply:
From the changelog, it looks like we're using the semconv spec version as the version number for this gem, so I'm inclined to continue that. One option is to leave the version at 1.10.0, fix the changelog, and actually release 1.10.0 (the only change in the meantime was to fix a build issue). Otherwise we can continue with the bump to 1.10.1 (and still fix the changelog). Any preference?

Reply:
I reverted the 1.8.0 -> 1.10.0 change in the changelog, so we just need to decide what we want to use for the version number for _this_ release.

Reply:
I went with 1.10.0 🤷